system-tests/src/tests/history/history.spec.ts

```diff
@@ -97,7 +97,7 @@ test("test", async ({ page, headless }) => {
 
   // Click text=Save
   await page.click("text=Save")
-  await page.waitForTimeout(100)
+  await page.waitForTimeout(200)
```

Comment:
https://github.com/rage/secret-project-331/blob/db862a5090fd031fe944693190ff5a77054decf8/system-tests/src/tests/creating-course-and-returning-an-exercise.spec.ts#L147